.github/workflows/main.yml

```
@@ -72,6 +72,16 @@ jobs:
             os: windows-2019
             rust: stable-x86_64
             target: x86_64-pc-windows-msvc
+          - build: windows-clang
+            os: windows-2019
+            rust: stable
+            target: x86_64-pc-windows-msvc
+            env: $Env:AR="llvm-ar" ; $Env:CC="clang" ; $Env:CXX="clang++" ; $Env:RUSTFLAGS="-Clinker=lld-link" ;
+          - build: windows-clang-cl
+            os: windows-2019
+            rust: stable
+            target: x86_64-pc-windows-msvc
+            env: $Env:AR="llvm-ar" ; $Env:CC="clang-cl" ; $Env:CXX="clang-cl" ; $Env:RUSTFLAGS="-Clinker=lld-link" ;
```

If you write it like this:

```suggestion
            env: AR=llvm-ar\nCC=clang-cl\nCXX=clang-cl\nRUSTFLAGS=-Clinker=lld-link
```

Then use:

```
 - run: echo -e "${{ matrix.env }}" >> "$GITHUB_ENV"
   shell: bash
```

Then I think it will work